src/Files.App/ViewModels/Previews/ImagePreviewViewModel.cs

```diff
@@ -20,14 +20,18 @@ public ImageSource ImageSource
 			private set => SetProperty(ref imageSource, value);
 		}
 
-		public ImagePreviewViewModel(ListedItem item) : base(item) { }
+		public ImagePreviewViewModel(ListedItem item)
+			: base(item)
+		{
+		}
 
 		public static bool ContainsExtension(string extension)
 			=> extension is ".png" or ".jpg" or ".jpeg" or ".bmp" or ".gif" or ".tiff" or ".ico" or ".webp";
```

Comment:
Same here, we should use the existing helper. Can add a TODO so we don't forget.